apps/dcos_net/src/dcos_net_mesos.erl

```diff
@@ -158,3 +176,22 @@ maybe_fatal_error({failed_connect, Info}) ->
     end;
 maybe_fatal_error(_Error) ->
     ok.
+
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_summary:new([
```

Comment:
Please add `type` tag (from `Request`) to all of these metrics.

Reply:
I am not sure I understand what `type` tag means or what `Request` you mean. Could you please clarify?

Reply:
```erlang
call(Request, HTTPOptions, Opts) ->
    Type = maps:get(type, Request),
```